TypeCobol.Test/Parser/Programs/Cobol85/SpecialNamesParagraph.rdz.cbl

```diff
@@ -4,12 +4,28 @@
        CONFIGURATION SECTION.
        special-names.
            decimal-point comma
+      
+           ALPHABET alphabet1 IS STANDARD-1
+           ALPHABET alphabet2 IS STANDARD-2
+           ALPHABET alphabet3 IS NATIVE
+           ALPHABET alphabet4 IS EBCDIC
+           ALPHABET alphabet5 IS SPACE
+           ALPHABET alphabet6 IS LOW-VALUE THROUGH HIGH-VALUE
+           ALPHABET alphabet7 IS ZERO THRU QUOTE
+           ALPHABET alphabet8 IS SPACE ALSO QUOTE
+           ALPHABET alphabet9 IS SPACE ALSO QUOTE ALSO ZERO
+      
            SYMBOLIC CHARACTERS LOW-VAL IS 1
                                HEX-15  IS 22
                                HI-VAL  IS 256
+      
            CLASS NAMECHAR IS 'A' THRU 'I'
                              'J' THRU 'R'
                              'S' THRU 'Z'
+           CLASS class1 IS LOW-VALUE THROUGH HIGH-VALUE
+           CLASS class2 IS SPACE
+           CLASS class3 QUOTE
+           CLASS class4 IS ZERO
```

Comment:
This test is already quite exhaustive, but could we also add the following ?
```cob
           CLASS class5 IS LOW-VALUE QUOTE SPACE HIGH-VALUE ZERO
```

Reply:
done in 0a812376dc42e6abb0e363640ad529c11d9a3c86.